.github/workflows/cli-release-process.yml

```
@@ -19,7 +19,10 @@ env:
   JAVA_DISTRO: temurin
   NEXT_VERSION: '1.0.0-SNAPSHOT'
   GRAALVM_VERSION: '22.1.0'
-  PACKAGE_TYPE: 'uber-jar'
+  MVN_PACKAGE_TYPE: 'uber-jar'
+  NPM_PACKAGE_NAME: 'dotcli-test'
```

```suggestion
  NPM_PACKAGE_NAME: 'dotcli'
```

Are we sure we want to call it `test`?

---

You are right, it's a mistake.